.docs/user-guide/automation.md

```diff
@@ -14,7 +14,32 @@ orchestration tools below.  Optionally, Docker is also listed in some examples.
 
 <br>
 ## Ansible
-ToDo
+With Ansible, installing the latest REX-Ray binaries can be accomplished by
+including the `codenrhoden.rexray` role from Ansible Galaxy.  The role accepts
+all the necessary variables to properly fill out your `config.yml` file.
```

Comment:
Hi @codenrhoden,

All, see, you presume that I know what the hell about which I'm talking. Or that I think before I speak. I understand now. This looks fine then. I thought the YAML you listed was going **into** the REX-Ray config file, and I was concerned about the `hosts`, `sudo`, and `roles` keys at the root of the config. I understand better now what is occurring. Thank you very much for taking the time to explain it for me.